source/making_payments/index.html.md.erb

```diff
@@ -5,120 +5,139 @@ weight: 70
+Securely generated UUIDs are not guessable, so malicious users will
+not be able to alter the payment ID and intercept other users’ payments.
 
->Note: If you create an ID yourself, you’ll likely need to store this in a
-datastore mapped to the payment ID just after you create a payment.
+If you create an ID yourself, you will likely need to store this in a datastore mapped
+to the payment ID just after you create a payment.
 
 ## Accepting returning users
 
- A user directed to the return URL could have:
+ A user directed to the `return_url` could have:
 
  - paid successfully
- - not paid because their card was rejected or they clicked cancel
- - not paid because your service (via the API) or service manager (via the self-service dashboard) cancelled the payment in progress
+ - not paid because their card was rejected or they selected __Cancel__
+ - not paid because your service (via the API) or service manager (via the GOV.UK Pay admin tool) cancelled the payment in progress
  - not paid because of a technical error
 
 Your service should use the API to check the payment status when the user
-reaches the return URL, and provide an appropriate response based on the final
+reaches the `return_url`, and provide an appropriate response based on the final
 status of the payment attempt.
 
 ## When a user does not complete their payment journey
 
-The user may close their browser or lose internet connection in the middle of
-the payment flow on GOV.UK Pay. These users will not be redirected back to
-your service.
+Users may close their browser or lose internet connection in the middle of
+their payment journey on GOV.UK Pay. In this event, users will not be
+redirected back to your service.
 
-You can still check on the status of these payments by making a GET request
-using the Location Header or Self Link, the same way you would if they were
-redirected, but just after a set time (eg, an hour).
+You can still [check on the status of these payments](#making-payments), but
+after a set time (eg, an hour).
 
->Note: GOV.UK Pay will expire incomplete payments after 90 minutes, but you should expect an occasional success or failure if the user experienced problems right at the moment of the redirect.
+GOV.UK Pay will expire incomplete payments after 90 minutes, but you should
+expect occasional notifications of success or failure if users experienced
+problems right at the moment of the redirect.
 
-If a user does not have enough funds in their account to make a payment, the
-current GOV.UK Pay frontend will not let them try again with separate card
-details. They will have to return to the service to retry the payment.
+If a user does not have enough funds in their account to make a payment,
+GOV.UK Pay will not allow them to attempt the payment again with alternative
+card details. Instead, in this situation the user would need to return to the
+service to retry the payment.
```

Comment:
This applies more generally than just not having enough money: it’s the same if the card is declined for any reason (e.g. incorrect address or expiry date).